html/index.php

```diff
@@ -0,0 +1,67 @@
+<?php
+header('Access-Control-Allow-Origin: *');
+header('Access-Control-Allow-Headers: *');
+header('Access-Control-Allow-Methods: *');
+
+// jsonを返して死ぬやつ
+function sendResponse($obj)
+{
+    echo json_encode($obj);
+    die();
+}
+
+// DBに接続する
+try {
+    $db = new PDO('mysql:dbname=sns_api;host=mysql;charset=utf8', 'root', 'root');
+} catch (PDOException $e) {
+    echo json_encode('DB接続エラー:' . $e->getMessage());
+    die();
+}
```

Comment:
今はパラメタのエラー判定に引っかかってDB接続不要なリクエストの場合も必ずDB接続してしまっている。
シングルトンインスタンスのpdoをDB接続の都度参照した方が無駄な処理がない。

```suggestion
class Db
{
    private static $pdo = null;

    public static function getPdo()
    {
        if (null === self::$pdo) {
            /* pdoがないので作る */
            try {
                self::$pdo = new PDO('mysql:dbname=sns_api;host=mysql;charset=utf8', 'root', 'root');
            } catch (PDOException $e) {
                echo json_encode('DB接続エラー:' . $e->getMessage());
                die();
            }
        }
        return self::$pdo;
    }
}
```